Add DragonDrop

tresni opened the pull request:
Support for [DragonDrop](http://shinyplasticbag.com/dragondrop/) based on #3570

vitorgalvao commented on Casks/dragondrop.rb:
```diff
@@ -0,0 +1,11 @@
+  url 'http://shinyplasticbag.com/dragondrop/dragondrop.dmg'
+  name 'DragonDrop'
+  homepage 'http://shinyplasticbag.com/dragondrop/'
+  license :freemium
```
Isn’t it a timed trial?

tresni replied:
Yeah good point. I can change to commercial since it does stop functioning after the trial ends. 

Sent from my iPhone

> On Apr 23, 2015, at 5:26 PM, Vítor Galvão <notifications@github.com> wrote:
> 
> In Casks/dragondrop.rb:
> 
> > @@ -0,0 +1,11 @@
> > +cask :v1 => 'dragondrop' do
> > +  version :latest
> > +  sha256 :no_check
> > +  
> > +  url 'http://shinyplasticbag.com/dragondrop/dragondrop.dmg'
> > +  name 'DragonDrop'
> > +  homepage 'http://shinyplasticbag.com/dragondrop/'
> > +  license :freemium
> Isn’t it a timed trial?
> 
> —
> Reply to this email directly or view it on GitHub.
>

vitorgalvao replied:
Please do.

vitorgalvao commented:
There’s trailing whitespace that needs to be removed.

tresni commented:
@vitorgalvao Latest commit passes travis CI tests and addresses issues you've noted.  Let me know if there is anything else you think needs to happen.

vitorgalvao commented:
Just need you to please [squash your commits](http://davidwalsh.name/squash-commits-git), and it’s good to go.

tresni commented:
Sorry somehow managed to fuck up the initial rebase... This should take care of it.
